scripts/test-docker

```diff
@@ -58,10 +60,10 @@ for DOCKER_VERSION_SCRIPT in $(find ../dist/ -mindepth 1 -type f -exec basename
 					COMMAND="echo '' > /usr/bin/systemctl && dnf install -y yum && bash /install.sh && bash -c 'if [[ \"\$(docker version --format '{{.Client.Version}}') 2>/dev/null\" != ${DOCKER_VERSION}* ]]; then echo \"Found  version (\$(docker version --format '{{.Client.Version}}')) does not match \"${DOCKER_VERSION}\"\"; exit 1; fi'"
 				fi
 				docker run -d --privileged \
-                                        --label install-docker="true" \
```

Comment:
Maybe this is some auto formatting but it seems to break the alignment under `docker run -d --privileged`

Reply:
Fixed